packages/cli/src/commands/dev/dev.ts

```diff
@@ -22,11 +23,56 @@ interface HTTPSOptions {
   cert: Buffer;
 }
 
+interface PackageJson {
+  dependencies?: Record<string, string>;
+  devDependencies?: Record<string, string>;
+}
+
+interface MastraPackageInfo {
+  name: string;
+  version: string;
+}
+
+function getResolvedVersion(rootDir: string, packageName: string, specifiedVersion: string): string {
```

Comment:
In the deployer package we already use https://github.com/antfu-collective/local-pkg to do this, let's also use it in the CLI